acme-vendor/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const io = require('socket.io-client');
+const capsSocket = io.connect('http://localhost:3000/caps');
+
+capsSocket.emit('getAll', { queueId: 'acme-widgets' }); //was just 'driver'
+capsSocket.on('connect', () => {
+  console.log('acme-widgets connected to CAPS hub');
+
+  capsSocket.emit('join', 'acme-widgets');
+  capsSocket.emit('get-all-delievered-orders', {store: 'acme-widgets'});
+});
+
+capsSocket.on('disconnect', () => {
+  console.log('acme-widgets disconnected from CAPS hub');
+});
+
+
+capsSocket.on('delivered', (payload) => {
+  console.log(`Thank you for your order ${payload.customer}`);
+});
+
+// Simulating new customer orders every 5 seconds
+setInterval(() => {
+  const order = {
+    store: 'acme-widgets',
+    queueId: 'acme-widgets',
```

Comment:
The vendor needs to put things in the driver queue, because that's who it's being sent to. The driver will use the store property to determine which queue it needs to send its messages to.